packages/api-client/src/components/AddressBar/AddressBar.vue

```diff
@@ -141,11 +141,11 @@ function updateRequestPath(url: string) {
     <div class="m-auto flex flex-row items-center">
       <!-- Address Bar -->
       <div
-        class="addressbar-bg-states text-xxs p-0.75 group relative order-last flex w-full max-w-[calc(100dvw-24px)] flex-1 flex-row items-stretch rounded-lg lg:order-none lg:min-w-[580px] lg:max-w-[580px] xl:min-w-[720px] xl:max-w-[720px]">
+        class="address-bar-bg-states text-xxs p-0.75 group relative order-last flex w-full max-w-[calc(100dvw-24px)] flex-1 flex-row items-stretch rounded-lg lg:order-none lg:min-w-[580px] lg:max-w-[580px] xl:min-w-[720px] xl:max-w-[720px]">
         <div
           class="pointer-events-none absolute left-0 top-0 block h-full w-full overflow-hidden rounded-lg border">
           <div
-            class="bg-mix-transparent bg-mix-amount-90 z-context absolute left-0 top-0 h-full w-full"
+            class="bg-mix-transparent bg-mix-amount-90 absolute left-0 top-0 z-[1002] h-full w-full"
```

Comment:
why z1002? shouldn't we have set values for z-index?

Reply:
exception, otherwise the fader are going over - we'll dig this more

Reply:
There is a z-context-plus if you want to try that

Reply:
Oh wait it's being used below, z-context-plus-2? 😓

Reply:
yes - because the way we display address server and history menu - can be enhanced!